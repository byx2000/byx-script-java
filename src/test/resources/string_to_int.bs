function stringToInt(s) {
    var result = 0
    for (var i = 0; i < s.length(); ++i) {
        var d = s.codeAt(i) - '0'.codeAt(0)
        result = result * 10 + d
    }
    return result
}

Console.println(stringToInt('0'))
Console.println(stringToInt('2'))
Console.println(stringToInt('123'))
Console.println(stringToInt('32453532'))
