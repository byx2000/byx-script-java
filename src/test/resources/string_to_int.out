0
2
123
32453532
